service/src/main/kotlin/io/provenance/api/frameworks/objectStore/ObjectStoreService.kt

```diff
@@ -1,57 +1,82 @@
 package io.provenance.api.frameworks.objectStore
 
-import com.google.protobuf.Message
 import io.provenance.api.domain.objectStore.ObjectStore
-import io.provenance.api.frameworks.config.ObjectStoreConfig
+import io.provenance.api.frameworks.config.ObjectStoreProperties
 import io.provenance.api.models.eos.store.StoreProtoResponse
 import io.provenance.api.models.eos.store.toModel
+import io.provenance.api.util.toModel
 import io.provenance.scope.encryption.crypto.Pen
 import io.provenance.scope.encryption.domain.inputstream.DIMEInputStream
 import io.provenance.scope.encryption.ecies.ProvenanceKeyGenerator
 import io.provenance.scope.encryption.model.DirectKeyRef
 import io.provenance.scope.objectstore.client.OsClient
-import java.io.InputStream
+import java.io.ByteArrayInputStream
+import java.security.KeyPair
 import java.security.PrivateKey
 import java.security.PublicKey
+import java.time.Duration
+import java.time.OffsetDateTime
 import java.util.concurrent.TimeUnit
 import org.springframework.stereotype.Component
+import tech.figure.objectstore.gateway.client.GatewayClient
+import tech.figure.objectstore.gateway.client.GatewayJwt
 
 @Component
 class ObjectStoreService(
-    private val osConfig: ObjectStoreConfig
+    private val osConfig: ObjectStoreProperties,
```

Comment:
nit: change the name to match - i.e., `osProperties` or `osProps`

Reply:
sure